src-tauri/src/northstar/profile.rs

```diff
@@ -0,0 +1,36 @@
+use crate::GameInstall;
+
+#[tauri::command]
+pub fn fetch_profiles(
+    game_install: GameInstall,
+) -> Result<Vec<String>, String> {
+
+    let mut profiles: Vec<String> = Vec::new();
+
+    let pattern = format!("{}/*/mods", game_install.game_path);
+    dbg!(&pattern);
+    for e in glob::glob(&pattern).expect("Failed to read glob pattern") {
+        let path = e.unwrap();
+        let mut ancestors = path.ancestors();
+
+        ancestors.next();
+
+        let profile_path = std::path::Path::new(ancestors.next().unwrap());
```

Comment:
Right now I only check for `/mods`, What are some other things to check for?

Right now I got:
- /mods
- /plugins
- /packages
- /Northstar.dll

Maybe the following?:
- logs
- runtime
- enabledmods.json
- placeholder.playerdata.pdata

Reply:
Yeah, I'd say if any of those files are in there, treat it as a profile. Given that technically any folder could be a profile the condition for a profile more or less is "looks vaguely like it belongs to Norhtstar" and "isn't one of the game's existing folders like `r2` etc ^^

Maybe @BigSpice can tell us what he did for profile detection in VTOL :eyes: